src/components/flashcard/Flashcard.css

```diff
@@ -0,0 +1,47 @@
+.card {
+  width: 300px;
+  height: auto;
+  border: 2px solid #3498db;
+  border-radius: 8px;
+  perspective: 1000px;
```

Comment:
I've never seen the `perspective` css prop before! ChatGpt says it's generally used along with 3D transformations to set the distance of the Z axis from the user, so maybe not what we want here. Also, I tried changing the number and it doesn't seem to have any effect on the appearance of the card.

I think you might be wanting box-shadow instead. 
use this and see if it's what you were wanting:
`box-shadow: 4px 4px 8px rgba(0, 0, 0, 0.2);`

a short explanation from ChatGpt: The box-shadow property in CSS allows you to add a shadow effect to an element. It takes a set of values that define the offset, blur radius, spread radius, color, and inset property of the shadow.

Reply:
I agree. Perspective is also used when CSS is interactive. So, since we were not sure whether the card would visually flip, I played with this parameter a bit. For now, it's not relevant. Thanks!